packages/sdks/src/functions/get-block-styles.ts

```diff
@@ -23,25 +23,8 @@ const getStyleForTarget = ({
       };
     }
     default:
-      return {
-        ...(styles.large ? convertStyleObject(styles.large) : {}),
-
-        ...(styles.medium
-          ? {
-              [getMaxWidthQueryForSize('medium')]: convertStyleObject(
-                styles.medium
-              ),
-            }
-          : {}),
-
-        ...(styles.small
-          ? {
-              [getMaxWidthQueryForSize('small')]: convertStyleObject(
-                styles.small
-              ),
-            }
-          : {}),
-      };
+      // By default we do not want inline styles
+      return {};
```

Comment:
I simplified all this logic recently to only run on React Native, so you can take my changes from the merge conflict.